### DISCUSSION TITLE
Feedback Value & Custom Images

### DISCUSSION THREAD

**Discussion by SkillenUK** [nguyenquyhy/Flight-Tracker-StreamDeck#421] [Q&A]
Hi, I think that the feedback values triggering the custom images are not working for me. If anyone can point me in the right direction I'd appreciate it.

I've installed Flight Tracker 0.52 from the Stream Deck store and I have MSFS2020 1.36.2.0.

I've been playing around with it a lot, I feel like the following Generic Toggle Button configuration for toggling the approach button should work...

![image](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/assets/25122561/11734f1e-a2b6-4b21-ac60-2757387fb4c1)

It does engage and disengage the button, but it doesn't toggle my ON/OFF images based on the Feedback Value. If I output the feedbsck value to to a display value then it remains at zero. So clearly I have either misunderstood the use of the variables or I have selected the wrong one. I have tried to configure a handful of buttons but have not had any success so far.

Likewise the following configuration for the auto-throttle doesn't work in the same way...

![image](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/assets/25122561/de574bed-bb94-4583-a4b6-164a45ef665a)

I'm trying to setup some buttons to perform some of the autopilot functions in the FlyByWire A320, any guidance for getting the feedback values working would be appreciated. Thanks,

Skillen.

**Comment by JGA09** [comment]
Did you find the solution to your problem? I'm trying to do the same thing without success. It only displays the same yellow bars as the preset toggle buttons.

**Reply by bobjs1** [reply]
Me as well
